assets/app/features/upgrade/templates/landing-page.jade

```diff
@@ -1,39 +1,57 @@
-            strong(translate='') upgrade.steps.landing.note
+                    span(translate='') upgrade.steps.landing.prechecks.form.check-again
 
-    button.btn.btn-success.center-block(
-        ng-disabled="!upgradeLandingVm.prechecks.completed || !upgradeLandingVm.prechecks.valid || upgradeLandingVm.prepare.running",
-        ng-class="{'spinner-visible': upgradeLandingVm.prepare.spinnerVisible, active: upgradeLandingVm.prepare.running}",
-        ng-click="upgradeLandingVm.beginUpgrade()"
-    )
-        suse-lazy-spinner(delay="2000", active="upgradeLandingVm.prepare.running",
-            visible="upgradeLandingVm.prepare.spinnerVisible")
-        span(translate='') upgrade.steps.landing.form.button
+            .upgrade.pull-left(ng-class="{'active': upgradeLandingVm.mode.valid, 'inactive': !upgradeLandingVm.mode.valid}")
+                crowbar-circle(ng-class="{'active': upgradeLandingVm.mode.valid}") 3
+                .h4.text-center(translate='') upgrade.steps.landing.upgrade.title
+                p(translate='') upgrade.steps.landing.upgrade.description
+                button.btn.btn-primary(
+                    ng-disabled="!upgradeLandingVm.prechecks.completed || !upgradeLandingVm.prechecks.valid || upgradeLandingVm.prepare.running || !upgradeLandingVm.mode.valid",
+                    ng-class="{'spinner-visible': upgradeLandingVm.prepare.spinnerVisible, active: upgradeLandingVm.prepare.running}",
+                    ng-click="upgradeLandingVm.beginUpgrade()"
+                )
+                    suse-lazy-spinner(delay="2000", active="upgradeLandingVm.prepare.running",
+                    visible="upgradeLandingVm.prepare.spinnerVisible")
+                    span(translate='') upgrade.steps.landing.form.button
+        .col-md-6
+            .mode.pull-left(ng-class="{'active': upgradeLandingVm.prechecks.valid, 'inactive': !upgradeLandingVm.prechecks.valid}")
+                crowbar-circle(ng-class="{'active': upgradeLandingVm.mode.active}") 2
+                .h4.text-center(translate='') upgrade.steps.landing.mode.title
+                .initial
+                    p(translate='', ng-if='upgradeLandingVm.mode.type == false') upgrade.steps.landing.mode.description
+                .normal
+                    strong
+                        p(translate='', ng-if='upgradeLandingVm.mode.type == "disruptive"') upgrade.steps.landing.mode.normal.title
+                    button.btn.btn-primary(
+                        ng-disabled="upgradeLandingVm.mode.type != 'disruptive' || upgradeLandingVm.mode.valid",
+                        ng-if="upgradeLandingVm.mode.type == 'disruptive'"
+                        ng-click="upgradeLandingVm.continueNormal()"
+                    )
+                        span(translate='') upgrade.steps.landing.mode.normal.button
+                    p(translate='', ng-if='upgradeLandingVm.mode.type == "disruptive"') upgrade.steps.landing.mode.normal.description1
```

Comment:
Please, use a container and only one ng-if to avoid unnecessary watchers. Same on line 55
Also use `===`